core-v1/src/main/scala/algebra/Expr.scala

```diff
@@ -1039,6 +1044,17 @@ object Expr {
       copy(debugging = debugging)
   }
 
+  final case class Repeat[-I, +O, OP[_]](
+    inputExpr: Expr[I, O, OP],
+    override private[v1] val debugging: Debugging[Nothing, Nothing] = NoDebugging,
+  )(implicit
+    opO: OP[IterableOnce[O]],
+  ) extends Expr[I, IterableOnce[O], OP]("repeat") {
+    override def visit[G[-_, +_]](v: Visitor[G, OP]): G[I, IterableOnce[O]] = v.visitRepeat(this)
+    override private[v1] def withDebugging(debugging: Debugging[Nothing, Nothing]): Repeat[I, O, OP] =
+      copy(debugging = debugging)
+  }
```

Comment:
BLKR: This should have documentation. Also, maybe it would be better to put some kind of limit to avoid an infinite loop.